ingest/ingest-datasets.py

```diff
@@ -1,148 +1,72 @@
-    #   Arguments:
-    #   --threads: number of threads to use (default = 8)
-    #   --es', elasticsearch service URL (default="http://localhost:9200")
-    #   --publish', publish to elasticsearch? (default=False)
-    #   --rebuild', rebuild elasticsearch index? (default=False)
-    #   --mapping', dataset elasticsearch mapping document (default="mappings/dataset.json")
-    #   --sparql', sparql endpoint (default='http://deepcarbon.tw.rpi.edu:3030/VIVO/query')
-    #   [out]: file name of the elasticsearch bulk ingest file
-    # Example:
-    #   python3 ingest-datasets.py output4 --threads 2 --mapping mappings/XXXXX.json ...
-
-    Main(get_objects_query=get_datasets_query, create_object_doc_function=create_dataset_doc,
-         object_index=_index, object_type=_type)
-
-    ################################################################################################################
-    #
-    # SOME RUNNING SCRIPTS:
-    #
-    # To start elastic search: [elastic search folder]/bin/elasticsearch
-    #
-    # 0) (HAVE CAUTION.) Delete existing data to avoid uploading error due to mismatching:
-    #       (For localhost) curl -XDELETE 'localhost:9200/dco/dataset'
-    #       (For dcotest)   curl -XDELETE 'dcotest.tw.rpi.edu:49200/dco/dataset'
-    #
-    #
-    # 1) Manually upload mapping:
-    #       (For localhost) curl -XPUT 'localhost:9200/dco/dataset/_mapping?pretty' --data-binary @mappings/dataset.json
-    #       (For dcotest)   curl -XPUT 'dcotest.tw.rpi.edu:49200/dco/dataset/_mapping?pretty' --data-binary @mappings/dataset.json
-    #
-    # 2) Generate bulk data:
-    #       python3 ingest-datasets.py output#
-    #   and then upload bulk data manually
-    #       (For localhost) curl -XPOST 'localhost:9200/_bulk' --data-binary @output#
-    #       (For dcotest)   curl -XPOST 'dcotest.tw.rpi.edu:49200/_bulk' --data-binary @output#
-    #
-    # 2') Generate bulk data and upload bulk data automatically:
-    #       (For localhost) python3 ingest-datasets.py --es 'localhost:9200' --publish output#
-    #       (For dcotest)   python3 ingest-datasets.py --es 'dcotest.tw.rpi.edu/search/' --publish output#
-    #
-    # 3) To view and operate in Sense:
-    #       GET dco/dataset/_mapping
-    #       GET dco/dataset/_search
-    #       DELETE /dco/dataset/
-    #       DELETE /dco/dataset/_mapping
-    #       and etc...
-    #################################################################################################################
\ No newline at end of file
+    Main(get_objects_query_location=GET_OBJECTS_QUERY_LOCATION,
+         describe_object_query_location=DESCRIBE_DATASET_OBJECT_LOCATION,
+         create_object_doc_function=create_dataset_doc,  # Need to create case-varying create_xxxxxx_doc in helper file
+         object_index=INDEX, object_type=TYPE,
+         variable_name_sparql=VARIABLE_NAME_SPARQL)
+
+# Finally, run the ingest process. See explanations and examples below.
+
+# Usage:
```

Comment:
Yes I think it's better that these should be in a README

On Friday, October 23, 2015, Patrick West <notifications@github.com> wrote:

> In ingest/ingest-datasets.py
> <https://github.com/tetherless-world/dco-elasticsearch/pull/29#discussion_r42831500>
> :
>
> > \ No newline at end of file
> > +    Main(get_objects_query_location=GET_OBJECTS_QUERY_LOCATION,
> > +         describe_object_query_location=DESCRIBE_DATASET_OBJECT_LOCATION,
> > +         create_object_doc_function=create_dataset_doc,  # Need to create case-varying create_xxxxxx_doc in helper file
> > +         object_index=INDEX, object_type=TYPE,
> > +         variable_name_sparql=VARIABLE_NAME_SPARQL)
> > +
> > +# Finally, run the ingest process. See explanations and examples below.
> > +
> > +# Usage:
>
> From line 32 to 72 these are general comments about the ingest process.
> General usage, sample command line for executing commands against
> elasticsearch.
>
> Do you think it would be a good idea to create a README with this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tetherless-world/dco-elasticsearch/pull/29/files#r42831500>
> .
>